app/models/checkout.rb

```diff
@@ -51,6 +51,24 @@ def self.longest_checkout_time_today(offset)
     Time.at(difference).utc.strftime('%H:%M:%S')
   end
 
+  def self.longest_checkout_game_today(offset)
+    start_time = (Time.now - 15.hours).strftime('%Y-%m-%d %H:%M:%S')
+    min_checkout = self.where(closed: false).where(
+                       "(check_out_time - '#{offset} hours'::interval) > ?",
```

Comment:
This is a large hole for SQL injection. Offset needs to be sanitized - change this to `Checkout.connection.quote(offset)` where `offset` currently is.